src/Paramore.Brighter.MessagingGateway.Kafka/KafkaMessageConsumer.cs

```diff
@@ -470,7 +470,7 @@ private void CommitAllOffsets(DateTime flushTime)
                     var offsets = listOffsets.Select(tpo =>
                         $"Topic: {tpo.Topic} Partition: {tpo.Partition.Value} Offset: {tpo.Offset.Value}");
                     var offsetAsString = string.Join(Environment.NewLine, offsets);
-                    s_logger.LogInformation($"Sweeping offsets: {Environment.NewLine} {{Offset}}", offsetAsString);
+                    s_logger.LogInformation("Sweeping offsets: {0} {Offset}", Environment.NewLine, offsetAsString);
```

Comment:
This was deliberate, as in this case I think you do want `Environment.NewLine` in the `message` argument, and not captured as a named parameter with name `0`. Although the string interpolation will probably trigger analyzers.